Cargo.toml

```diff
@@ -10,6 +10,13 @@ description = "UNIC - Unicode and Internationalization Crates"
 categories = ["parsing", "rendering", "encoding", "development-tools"]
 readme = "README.md"
 
+exclude = [
+    "data/**/*.txt",
+    "tools/**/*.py",
+    ".cargo/*.sh",
```

Comment:
I would say lets keep it simple and ignore the whole dirs by `"data/**", "tools/**", ".cargo/**"`.